rpc/backend/sign_tx.go

```diff
@@ -106,15 +104,13 @@ func (b *Backend) SendTransaction(args evmtypes.TransactionArgs) (common.Hash, e
 	syncCtx := b.ClientCtx.WithBroadcastMode(flags.BroadcastSync)
 	rsp, err := syncCtx.BroadcastTx(txBytes)
 	if rsp != nil && rsp.Code != 0 {
+		if shouldSkip, msg := HandleBroadcastRawLog(rsp.RawLog, txHash); shouldSkip {
```

Comment:
We don't have a local TX tracker here, so we should just be returning the error until we implement that: https://github.com/cosmos/go-ethereum/blob/f70aaa8399ccee429804eecf3fc4c6fd8d9e6cab/eth/api_backend.go#L325-L341

I believe returning the error without a tx tracker is still considered EVM compatible, and we can create a separate issue to add one in.